SPECS/glib/glib.spec

```diff
@@ -1,23 +1,25 @@
 Summary:        Low-level libraries useful for providing data structure handling for C.
 Name:           glib
-Version:        2.58.0
-Release:        12%{?dist}
+Version:        2.60.1
+Release:        1%{?dist}
 License:        LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/System
```

Comment:
Nit: since groups are deprecated and do not bring any value, I suggest removing this entry.